js/spotify.js

```diff
@@ -1,12 +1,125 @@
 // API Docs at: 
 // https://developer.spotify.com/technologies/web-api/search/
 
+'use strict';
 
-function searchByArtist(keyword) {
-  var url = 'http://ws.spotify.com/search/1/artist.json?q='+keyword;
```

Comment:
Kill `var` everywhere. You don't have a need for it anymore. [Read this](https://medium.com/javascript-scene/javascript-es6-var-let-or-const-ba58b8dcde75#.a4shy0x86)